core/src/main/java/org/neo4j/ogm/metadata/FieldsInfo.java

```diff
@@ -104,6 +79,42 @@
         }
     }
 
+    private static String findTypeParameterDescriptor(Field field, ObjectAnnotations objectAnnotations) {
+
+        String typeParameterDescriptor = null;
+
+        final Type genericType = field.getGenericType();
+        if (genericType instanceof ParameterizedType) {
+            ParameterizedType parameterizedType = (ParameterizedType) genericType;
+            final Type[] actualTypeArguments = parameterizedType.getActualTypeArguments();
```

Comment:
I know that this is also not part of the PR but shouldn't we set all final local vars to `final` or non of them?